src/components/VideoItem/VideoItem.js

```diff
@@ -1,27 +1,21 @@
 import React from 'react';
-import styled from 'styled-components';
+import './VideoItem.styles.css';
 
-const VideoItemContainer = styled.div`
-    display: flex;
-    align-items: center;
-    cursor: pointer;
-    text-align: center;
-
-`;
-
-const Thumbnail = styled.img`
-    max-width: 200px;
-    max-height: 250px ;
-`;
-
-function VideoItem({id, title, thumbnail, description}){
+function VideoItem({id, title, thumbnail, description, setSelectedVideo}){
+    const selectVideo = (video) => setSelectedVideo(video);
     return(
-        <VideoItemContainer id={id} key={`video-item-container-${id}`}>
-            <Thumbnail key={`video-thumbnail-${id}`} src={thumbnail} alt={description} />
-            <div key={`video-title-container-${id}`}>
-                <div key={`video-title-${id}`}>{title}</div>
-            </div>
-        </VideoItemContainer>
+        <>
+            {
+                id !== undefined &&
```

Comment:
No need to validate for undefined since you are using the `and` operator it will render only if id has a value different than undefined or null
```suggestion
                id &&
```